.github/workflows/dockerize_production.yml

```diff
@@ -0,0 +1,49 @@
+name: dockerize-prod
+
+on:
+  pull_request:
+  push:
+    tags:
+      - "*"
+    branches:
+      - master
+    paths:
+      - src/khoj/**
+      - config/**
+      - pyproject.toml
+      - Dockerfile
+      - docker-compose.yml
```

Comment:
Should changes to the docker-compose trigger a new docker image build?

Reply:
No, the `docker-compose` is just for use outside of the application. I'll update this.